feedstock/recipe.py

```diff
@@ -40,9 +40,9 @@ def make_filename(variable, time):
 
 # Define the recipe
 
-target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
+target_chunks = {"time": 30} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
 
 recipe = XarrayZarrRecipe(
             file_pattern=pattern,
             target_chunks=target_chunks,
-            subset_inputs = {"time": 5}) # set 5 chunks per year, each with time of length 73 (to total 365)
\ No newline at end of file
+            subset_inputs = {"time": 13}) # with time chunks of 30, we need a total of 13 chunks (12x30-day chunks, 1x5-day chunk)
```

Comment:
```suggestion

```

Based on a closer reading of https://github.com/pangeo-forge/cesm-atm-025deg-feedstock/issues/5#issuecomment-1151287882, it's not actually clear to me that input subsetting is required for this recipe. Each NetCDF input is slightly less than 1 GB (according to the linked comment). Removing subset inputs will not necessarily fix the error we are seeing, but reducing then number of factors in play will help me focus on which aspect of the recipe may be causing the problem.

Reply:
@paigem, I'm going to accept this change and then run another test, to see if we get a different result.